opm/grid/common/GridPartitioning.cpp

```diff
@@ -391,7 +391,8 @@ void addOverlapLayer(const CpGrid& grid, int index, const CpGrid::Codim<0>::Enti
                 {
                     // Note: multiple adds for same process are possible
                     exportList.emplace_back(nb_index, owner, AttributeSet::copy);
-                    exportList.emplace_back(index, cell_part[nb_index],  AttributeSet::copy);
+                    // not need or check if
+                    //exportList.emplace_back(index, cell_part[nb_index],  AttributeSet::copy);
```

Comment:
Why is this not needed? Should it not be symmetrical?

Reply:
I do not understand this fully, but this will be done for all processes so this will be added when the other partition is processed anyway.